lib/brine.rb

```diff
@@ -1,10 +1,12 @@
-# brine.rb -- loader file for the rest of brine
+##
+# @file brine.rb
+# Loader file for the rest of brine.
```

Comment:
Starting to switch to Doxygen style since then my doc efforts can be more consistently pooled across projects.